pbkhrv/rtl_433-hass-addons
# Fix "-T option null" issue

## Summary

When #144 was merged I missed a check to see if device_topic_suffix is null. This causes issues #155 and #159 . This PR adds a simple config check and fixes the issue for me when I don't add this argument.


## Alternatives Considered

There are other PRs that also fix this issue but this is a simple one-liner. I also ported the changes to config.json from the next folder over to the main autodiscovery config.json so these options work once next is merged.

## Testing Steps

1. Run next without setting a value for device_topic_suffix
2. Add-On should start up
